src/flows/components/ClientList.tsx

```diff
@@ -14,7 +14,7 @@ const ClientList: FC = () => {
   const {launch} = useContext(PopupContext)
 
   return (
-    <FlexBox style={{flexWrap: 'wrap'}}>
```

Comment:
stupid question - why no move this to the class dfinition in css? is this a clockface req or something else?